LICENSE.md

```diff
@@ -1 +1,21 @@
-This work is licensed under the Creative Commons Attribution-ShareAlike 4.0 International License. To view a copy of this license, visit http://creativecommons.org/licenses/by-sa/4.0/ or send a letter to Creative Commons, PO Box 1866, Mountain View, CA 94042, USA.
+MIT License
+
+Copyright (c) 2020 Seth Cooper
```

Comment:
In the past we have used 'Copyright (c) 2020 Northeastern University, Boston, MA (USA).'